workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -195,14 +195,11 @@ async def get_regions_as_geojson() -> FeatureCollection:
     return feature_collection
 
 
-async def get_regions() -> List[tuple]:
+async def get_regions() -> List[dict]:
     query = "SELECT  name, ogc_fid FROM regions"
     async with get_connection() as conn:
-        record = await conn.fetch(query)
-    regions = []
-    for tuple_record in record:
-        regions.append(tuple(tuple_record))
-    return regions
+        records = await conn.fetch(query)
+    return [dict(r) for r in records]
```

Comment:
I thought about something like this, since the FIDs should be unique and suitable as keys. But your solution is fine as well.
```suggestion
        records = await conn.fetch(query)
    return {r["ogc_fid"]: r["name"] for r in records}
```

Reply:
we decided to keep the original suggestion in the PR